VnE/Source/Extensions/ChemPack/voronoi.py

```
@@ -0,0 +1,132 @@
+                    atoms[0].cif_cell_ga]
+        else:
+            ...
+
+        symms = [x[1] for x in sym_codes]
+        data = [tuple([x.atom_type, *list(x.cif_frac_coords)]) for x in atoms]
+        bools = [True if x in sel else False for x in atoms]
+
+        res = cpplib.VoronoiCalculation(cell, symms, data, bools, 6.0)
+        res['vertexes_dec'] = fracToDec(*cell, [list(x) for x in res['vertexes']])
+
+        points_list = point_class.PointsList(TREE_MODEL.getRoot(), name=f'Voronoi polyhedron')
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)